.grype.yml

```diff
@@ -0,0 +1,23 @@
+# List of vulnerabilities to ignore for the anchore scan
+# https://github.com/anchore/grype#specifying-matches-to-ignore
+# More info can be found in the docs/infra/vulnerability-management.md file
+
+# Please add whitelists in the following format to make it easier when checking
```

Comment:
Nit: Nava tends to use the word safelist instead of whitelist
```suggestion
# Please add safelists in the following format to make it easier when checking
```

Reply:
Gotcha, I'll have that in the next commit. I am going to make an example workflow that runs the scans in parallel